spark/v3.1/spark/src/main/java/com/netease/arctic/spark/writer/KeyedSparkBatchWrite.java

```diff
@@ -149,13 +168,15 @@ public DataWriterFactory createBatchWriterFactory(PhysicalWriteInfo info) {
 
     @Override
     public void commit(WriterCommitMessage[] messages) {
+      checkBlocker(tableBlockerManager);
       RewritePartitions rewritePartitions = table.newRewritePartitions();
       rewritePartitions.withTransactionId(txId);
 
       for (DataFile file : files(messages)) {
         rewritePartitions.addDataFile(file);
       }
       rewritePartitions.commit();
+      tableBlockerManager.release(block);
```

Comment:
It's better to release block like
```
try {
...
} finally {
     tableBlockerManager.release(block);
}
```

Reply:
If there is an exception during the commit phase, it will execute `abort()`, the release operation is also done in `abort()`

Reply:
I think release blocker in finally is better.  the commit action may also failed.